### DISCUSSION TITLE
如何标注SER与RE模型使用的数据

### DISCUSSION THREAD

**Discussion by TTTalen** [PFCCLab/PPOCRLabel#14] [Q&A] [answered]
官方数据，如以下所示。
zh_train_0.jpg   [{"transcription": "汇丰晋信", "label": "other", "points": [[104, 114], [530, 114], [530, 175], [104, 175]], "id": 1, "linking": []}, {"transcription": "受理时间:", "label": "question", "points": [[126, 267], [266, 267], [266, 305], [126, 305]], "id": 7, "linking": [[7, 13]]}, {"transcription": "2020.6.15", "label": "answer", "points": [[321, 239], [537, 239], [537, 285], [321, 285]], "id": 13, "linking": [[7, 13]]}]

问题与答案由linking字段表示。
在使用PaddleOCR如何才能实现这样子的效果

**Comment by GreatV** [comment]
目前还不行
可以参考一下这个项目完成关联
https://aistudio.baidu.com/projectdetail/4197468

**Reply by TTTalen** [reply, marked as answer]
好的谢谢，最终使用了标注时标注了_Q和_A后，对txt文件进行处理
